cas-server-core-api-ticket/src/main/java/org/jasig/cas/ticket/registry/TicketRegistryState.java

```diff
@@ -15,7 +15,7 @@
      * @return Number of ticket-granting tickets in the registry at time of invocation
      *         or {@link Integer#MIN_VALUE} if unknown.
      */
-    int sessionCount();
+    long sessionCount();
```

Comment:
No that won't work. Doesn't have to be a billion sessions; you just have to enable remember-me expiration policies, and CAS would bomb.